packages/server/graphql/mutations/helpers/notifications/SlackNotifier.ts

```diff
@@ -100,6 +107,61 @@ const makeEndMeetingButtons = (meeting: Meeting) => {
   }
 }
 
+const createLinkSectionContent = (
+  meetingId: string,
+  meetingUrl: string,
+  meetingShortLink?: string
+) => {
+  const shortLink = meetingShortLink || `https:/prbl.in/${meetingId}`
```

Comment:
-2 `prbl.in` goes to /invitation-link (please fix in other file, too)

Reply:
@mattkrick could you elaborate on this a bit more? previously link was created either using
```
makeSection(`*Link:*\n<${meetingUrl}|https:/prbl.in/${meeting.id}>`),
``` 
or
```
const maybeMeetingShortLink = makeAppURL(process.env.INVITATION_SHORTLINK!, `${meeting.id}`)
makeSection(`*Link:*\n<${meetingUrl}|${maybeMeetingShortLink}>`),
```
so unless I'm missing something obvious, I haven't changed anything in the logic here

Reply:
<img width="563" alt="Screenshot 2022-07-27 at 18 21 21" src="https://user-images.githubusercontent.com/1017620/181298785-00b55e8d-0ba3-4b1b-bee7-818e14b8dba8.png">

Seems like everything after `|` is just a mask for a real link (`meetingUrl`). The screenshot is from the current prod impl.

Reply:
If that's the case then prod is wrong :slightly_smiling_face: 

`prbl.in == action.parabol.co/invitation-link` (we picked `.in` because it's short for invitation :yum: )
from the screenshot it looks like the link is good, but the link label is bad. If that's the case, we can make it a friendly link label, or use [appOrigin](https://github.com/ParabolInc/parabol/blob/f221880dac24332874939b6c0a3f7328f4b76023/packages/server/appOrigin.ts) to make it the real URL.

or since there's a button that does the same thing as the link, maybe we just remove the url?

any of those work for me! Just feels weird having the link label look like a url, but be invalid

Reply:
it feels weird for sure, thank you for catching this! as this touches _every_ notification that we send, I don't want to decide on my own. I feel like having a visible link could be useful for easy copy/paste. on the other hand, having a long URL here also doesn't feel good. @enriquesanchez @ackernaut any thoughts?

Reply:
Yep, the intention for having the shortened URL there was to allow for easy copy/paste in case someone needed it to invite others. It's definitely a _nice-to-have_ and not critical, if there are strong tensions against it I don't have a preference.

That said, if we change it we should probably look at the other meeting notifications that also include the shortened link URL.

<img width="511" alt="Captura de Pantalla 2022-07-27 a la(s) 4 09 40 p m" src="https://user-images.githubusercontent.com/3276087/181372807-92726ae5-2612-4de5-bae9-b010af20a76e.png">

Reply:
The visible URL is short, but when you hover it, it’s long? Is that true? I would want them both to match. I like the short version. I agree the long version may be too much to print here.

Reply:
@ackernaut @enriquesanchez, sorry for not being clear enough. I like to short version too. the problem is, it's actually wrong. for example, I just copied the URL from our squad check-in meeting - https:/prbl.in/fiOGvDz7sA. when you try to open that URL, it'll say `invitation invalid` as we're trying to use the invitation URL format with `meetingId`. The question is, how do we want to fix that? It's not specific to this PR, all URLs we send in notifications on production are technically broken. they work if you click on them as we display the short URL only as a mask, the actual link is the long, correct URL. if you copy it by hand, it won't work. 

I also feel like it shouldn't block this issue. This PR doesn't introduce any regressions, and we really need to have it merged before the next release. @mattkrick are you ok with me creating a new issue for the URL problem and moving on with that PR?

Reply:
I see. If you didn’t make changes to the link part and want to move on, okay by me. I think the short URL should work and the format that’s printed in the view is also what would be copied and used in the browser (or typed out if you were to read it for some reason). We can aim for that AC in another issue, or we can just yank it from the design as it’s confusing and broken otherwise.

Reply:
I created a separate issue where we can discuss how to fix it - https://github.com/ParabolInc/parabol/issues/6968